lib/data_update_scripts/20210503175027_move_summary_websiteurl_and_location_to_profile.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class MoveSummaryWebsiteUrlAndLocationToProfile
+    def run
+      Profile.select(:id,
```

Comment:
Given that we have one profile per user, this is going to run for a long time. You may want to coordinate with @maestromac for running this. Maybe we shouldn't even make it a DUS but something that gets manually kicked off via  a Rake task or similar.

Reply:
I'm okay with this staying as DUS and will help to monitor it along the way.